### PR TITLE
modify encoding module

### PR DESCRIPTION
**Pull request by Leospard**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/triple/dubbo3_server.go [hunk 1]
```diff
@@ -46,6 +44,10 @@ import (
 	"github.com/dubbogo/triple/pkg/tracing"
 )
 
+var (
+	log = logger.LoggerWrapper{}
```
Comment (AlexStocks) [inline pkg/triple/dubbo3_server.go]: 没看出有加一个全局 log 对象的必要

Comment (AlexStocks) [inline pkg/triple/dubbo3_server.go]: 没看出有加一个全局 log 对象的必要

**Review by AlexStocks — APPROVED** [review]

**Review by justxuewei — APPROVED** [review]

**Comment by justxuewei** [comment]
Lgtm, thanks!

**Comment by justxuewei** [comment]
I think all changes should be merged into the main branch once optimization is done. Currently, the target branch is `triple-optimize`. @Leospard @AlexStocks

**Pull request merged**